File: Python/iniciante/1066paresimparespositivosnegativos.py
P = N = p = n = 0
for i in range(0, 5):
    n = float(input())
    if n % 2 == 0:
        P += 1
    else:
        N += 1
    if n > 0:
        p += 1
    if n < 0:
        n -= 1
print("%d valor(es) par(es)" % P)
print("%d valor(es) impar(es)" % N)
print("%d valor(es) positivo(s)" % p)
print("%d valor(es) negativo(s)" % n)

File: C/Iniciante/1066paresimparespositivosnegativos.c
#include <stdio.h>
int main() {
    int A,i,P=0,N=0,p=0,n=0;
    for(i=1;i<=5;i++){
        scanf("%d",&A);
        if(A%2==0){
            P++;
        }else{
            N++;
        }
        if(A>0)p++;
        if(A<0)n++;
    }
    printf("%d valor(es) par(es)\n",P);
    printf("%d valor(es) impar(es)\n",N);
    printf("%d valor(es) positivo(s)\n",p);
    printf("%d valor(es) negativo(s)\n",n);
    return 0;
}